LICENSE.md

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2015, Vinta Software Studio
+Copyright (c) 2013, Charles Denton and Marc Tamlyn
```

Comment:
I guess we could do what vlc does:
https://github.com/videolan/vlc/blob/master/THANKS